tests/vm_test.go

```diff
@@ -918,7 +924,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 						"New VMI was created, but StateChangeRequest was never cleared")
 				})
 
-				It("should not remove a succeeded VMI", func() {
+				It("[test_id:2190] should not remove a succeeded VMI", func() {
```

Comment:
I am missing the action start vm, login to VM and poweroff and check that vmi is not deleted. it will enter status.phase.succeeded

Reply:
Confirmed test is correct in offline conversation with @ILpinto